README.md

```diff
@@ -7,8 +7,12 @@ InstaB@ddies aims to create a simplified Instagram clone that allows users to vi
 * Comment on an uploaded picture
 
 ## Members :bust_in_silhouette:
+<<<<<<< HEAD
```

Comment:
Please remove lines 10 - 13 that contain <<< or === or >>>, those were demarcations put by your editor as a result of a merge and should not be visible.